### PR TITLE
Add Apache 2.0 license to pkg/ddc/alluxio/shutdown_test.go

### LINKED ISSUE #1161: [DOC] Add Apache 2.0 license to pkg/ddc/alluxio/shutdown_test.go
**Provide a link to that doc page:**
https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ddc/alluxio/shutdown_test.go
**What is the defect and your suggestions on improvement:**
There is no license statement for this file.
Apache 2.0 license should be added at the beginning.

### PR DESCRIPTION
**Pull request by Dice21**
### Ⅰ. Describe what this PR does
Add Apache 2.0 license to pkg/ddc/alluxio/shutdown_test.go.

### Ⅱ. Does this pull request fix one issue?

Fixes #1161 

### Ⅳ. Describe how to verify it
Check the beginning of pkg/ddc/alluxio/shutdown_test.go.

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/alluxio/shutdown_test.go [hunk 1]
```diff
@@ -1,3 +1,15 @@
+/*
```
Comment (cheyang) [inline pkg/ddc/alluxio/shutdown_test.go:1]: I suggest you should check the content of license header. thanks.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/alluxio/shutdown_test.go (hunk 1 above)
Comment (Dice21) [reply pkg/ddc/alluxio/shutdown_test.go:1]: The copyright part is missed and i added them.
Is it right now? Please check it again.

**Comment by cheyang** [comment]
@Dice21 I suggest you should fix the conflict.

**Comment by Dice21** [comment]
@cheyang Done.

**Comment by cheyang** [comment]
> @cheyang Done.

I think you have changed too many files. you only need to update one file.

**Comment by Dice21** [comment]
> > @cheyang Done.
> 
> I think you have changed too many files. you only need to update one file.

Oh i see. Please check it again:)

**Pull request merged**